examples/ethereum/list-rewards/main.go

```diff
@@ -11,56 +11,82 @@ import (
 	"log"
 	"time"
 
+	"google.golang.org/api/iterator"
+	"google.golang.org/protobuf/encoding/protojson"
+
 	"github.com/coinbase/staking-client-library-go/auth"
 	"github.com/coinbase/staking-client-library-go/client"
 	"github.com/coinbase/staking-client-library-go/client/options"
 	"github.com/coinbase/staking-client-library-go/client/rewards"
 	filter "github.com/coinbase/staking-client-library-go/client/rewards/rewardsfilter"
 	api "github.com/coinbase/staking-client-library-go/gen/go/coinbase/staking/rewards/v1"
-	"google.golang.org/api/iterator"
-	"google.golang.org/protobuf/encoding/protojson"
 )
-
-/*
- * Run the code with 'go run examples/ethereum/list-rewards/main.go' to view the rewards for the first validator on the Ethereum network.
- * Or, to view rewards for any arbitrary validator, simply replace the public key below.
- */
-
-const (
+ 
+ const (
 	apiKeyName    = "your-api-key-name"
-	apiPrivateKey = "your-api-private-key"
-
-	// https://beaconcha.in/validator/1
-	address = "0xa1d1ad0714035353258038e964ae9675dc0252ee22cea896825c01458e1807bfad2f9969338798548d9858a571f7425c"
-)
-
-func main() {
-	ctx := context.Background()
-
-	// Loads the API key.
-	apiKey, err := auth.NewAPIKey(auth.WithAPIKeyName(apiKeyName, apiPrivateKey))
-	if err != nil {
+    apiPrivateKey = "your-api-private-key"
```

Comment:
can you check indentation ?

Reply:
Yeah this part through me off, it looks good in my editor.

Reply:
tabs vs spaces